### PR TITLE
added addtl test for props for About component

### PR DESCRIPTION
**Pull request by leungnamson1023**
https://trello.com/c/BEsNF39g/28-test-about-component-to-make-sure-props-are-passed-down-small

### CODE REVIEW

**Review by tmkuba — COMMENTED** [review]
Hi Namson, the branch description and trello card indicates testing, but I'm not seeing that in the files changed?

File: client/src/app.test.js [hunk 1]
```diff
@@ -1,13 +0,0 @@
-import React from 'react';
```
Comment (matgbx) [inline client/src/app.test.js:1]: looks like the test was removed but a new one was not added?

**Review by matgbx — COMMENTED** [review]
Changes look good, but you seem to be lacking tests for the PR

File: client/src/app.test.js (hunk 1 above)
Comment (leungnamson1023) [reply client/src/app.test.js:1]: this was a duplicate test file. For some reason, it was inside my client/src folder when I already had one in my spec folder. So i deleted this file

**Comment by leungnamson1023** [comment]
Yeah, the test file that I removed was a duplicate file in my client/src folder when I already had one in my spec folder. Sorry for the confusion.

**Pull request closed without merging**